specs/Standards/NonFungibleToken/Event.md

```diff
@@ -0,0 +1,92 @@
+# Non-Fungible Token Event
+
+Version `1.0.0`
+
+## Summary
+
+Standard interfaces for NFT contract actions.
+
+## Motivation
+
+Nft driven apps such as marketplaces regularly perform many
+of the same acions such as `minting`, `burning` and `transfering`.
+Each app have their own way of performing these actions and it
+is difficult to consistently capture these events.
+This extension addresses that.
+
+For example, it's common in NFT marketplaces to have
+different methods of transfer a single token vs many 
+tokens. Other applications like wallets, will have
+a difficult time tracking consistently this information for many
+markets. This makes interactive with many nft driven apps 
+infeasible.
+
+Due to discussions here 
+https://github.com/near/NEPs/issues/254,
+a standard way to capture these events are needed.
+
+## Interface
+
+Many apps may use different mechanisms to perform their batching
+requirements. This interface standardises this process by capturing
+events of these activities through logs.
+
+```ts
+// Interface to capture an event
+// and return formatted event string.
+interface EventJsonLog {
+    // Takes `EventLogData` and returns
+    // `EVENT_JSON: <EVENT_LOG_DATA>`
+    (e:EventLogData):string
+}
```

Comment:
This notifies implementors of their need to produce the final string compatible with the indexer. Look at line 39 `EVENT_JSON {"standard":"nep171", ...}`.

This is the response to https://github.com/near/NEPs/issues/254#issuecomment-928249174 to capture all kinds of events not just json.